frontend/src/components/Dropdown/Dropdown.styled.js

```diff
@@ -0,0 +1,80 @@
+import styled from "styled-components";
+import {device} from "../../RWD/device";
+
+export const StyledDropdown = styled.div`
+  min-width: 160px;
+  padding: 0.5rem;
+  position: absolute;
+  top: calc(0% + .7rem);
+  right: 0;
+  background-color: #fff;
+  border: 1px solid #e1e1e1;
+  border-radius: 5px;
+  overflow: hidden;
+    @media ${device.tablet} {
+      min-width: 240px;
+      top: calc(0% + 1.4rem);
+    }
+    @media ${device.laptop} {
+      min-width: 300px;
+      top: calc(0% + 1.75rem);
```

Comment:
Nie korzystajmy z wartości względnych. Dodaj raczej wartości w "px". Ułatwi nam to refactoring